compiler/src/dmd/id.d

```diff
@@ -448,6 +448,8 @@ immutable Msgtable[] msgtable =
     { "outp"},
     { "outpl"},
     { "outpw"},
+    { "builtinsModuleName", "builtins" },
```

Comment:
note the conflict with importC's builtins.d druntime file; that one has a very unfortunate (and confusing) name...